.chloggen/1418-change-platform-to-api-detection.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Change the platform detection to be API detection. Previously, the operator was checking the plaform where it was running -OpenShift or Kubernetes-. Now, we just check if the OpenShift Route API is enabled"
```

Comment:
```suggestion
note: "Change the platform detection to be API detection. Previously, the operator was checking the plaform where it was running OpenShift or Kubernetes. Now, we just check if the OpenShift Route API is enabled"
```

Reply:
I think this change lefts the sentence with a different meaning or not well written. I would prefer to keep how it was written

Reply:
This does not need to be in the changelog. it's an internal refactoring.